spec/hardware/iommu.fmf

```diff
@@ -0,0 +1,31 @@
+summary: |
+    Select or provision a guest with the `Input–output memory management unit`.
+
+description: |
+    .. code-block::
+
+       iommu:
+           # Boolean, whether the guest supports IOMMU
+           is-supported: true|false
+           # String, the model name for IOMMU
+           model: "virtio"|"intel"|"smmuv3"|...
```

Comment:
I just realized we did not align this field with the rest of the specifications, see https://tmt.readthedocs.io/en/stable/spec/hardware.html#names-and-ids. This field should be called `model-name` as it's a label rather than a numerical ID.